engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -34,12 +45,6 @@
 import org.jboss.logging.annotations.Message;
 import org.jboss.logging.annotations.MessageLogger;
 
-import static org.jboss.logging.Logger.Level.DEBUG;
-import static org.jboss.logging.Logger.Level.ERROR;
-import static org.jboss.logging.Logger.Level.INFO;
-import static org.jboss.logging.Logger.Level.TRACE;
-import static org.jboss.logging.Logger.Level.WARN;
-
```

Comment:
Why I have the feeling the all our PR move the static imports up and down each time :)
(talking in general, no need to change)